src/infrastructure/validation/logger/LoggerManager.java

```diff
@@ -3,20 +3,56 @@
 /**
  * LoggerManager class that will be referenced by other modules,
  * for logging via the LoggerFactory
- * Implements ILogger interface and composes File and Console Loggers
+ * Implements ILogger interface and composes File and Console Loggers. 
+ * 
+ * Instantiation of FileLogger and ConsoleLogger indicated by,
+ * allowFileLogging and allowConsoleLogging,
+ * with defaults false, false.
  * 
  * @author Navaneeth M Nambiar
  */
 public class LoggerManager implements ILogger {
 
+	/** object that holds an instance of FileLogger */
+	private ILogger fileLogger;
+
+	/** object that holds an instance of ConsoleLogger */
+	private ILogger consoleLogger;
+	
+	/** 
+	 * boolean value indicating whether File Logger is created or not
+	 * defaults to false 
+	 */
+	private boolean allowFileLogging = false;
+	
+	/** 
+	 * boolean value indicating whether Console Logger is to be created or not
+	 * defaults to False 
+	 */
+	private boolean allowConsoleLogging = false;
```

Comment:
These maybe done later with the config file in the constructor with the loggerManager having the config file path as a variable

Reply:
ok but don't they need to be initialised to a default value?

Reply:
Agreed. A sensible default needs to be there.